DEVELOPMENT.md

```diff
@@ -0,0 +1,53 @@
+# FAQs
+
+## How do I set up Rust?
+
+[The official Rust docs guide](https://www.rust-lang.org/tools/install).
+
+## What IDE should I use?
+
+Most of this team uses VScode with the `rust-analyzer` extension but that preference is not hardcoded anywhere. Feel free to use any IDE you want!
+
+## What's a good first issue?
+
+We will try to keep a handful of [issues](https://github.com/stjude-rust-labs/wdl/issues) marked `good first issue` open and ready for new contributors.
+
+## I don't want to write code, can I still contribute?
+
+Sure!
+
+You can always open a [discussion](https://github.com/stjude-rust-labs/wdl/discussions/categories/rule-proposals) with a propsal for a new lint rule.
+
+We also appreciate feedback on our documentation. Feel free to look over any of our `*.md` files and note any issues you find. You can also explore our lint rule documentation by [installing `sprocket`](https://stjude-rust-labs.github.io/sprocket/installation.html) and reading the output of `sprocket explain`. (nota bene: we hope to replace `sprocket explain` with a website where each rule will have a dedicated page, but that has not been realized yet)
```

Comment:
I'm not sure I've ever seen anyone write out _nota bene_ before. Maybe just use the abbreviation _n.b._.

Reply:
😅 😅  my years taking Latin are showing